src/main/kotlin/Main.kt

```diff
@@ -25,10 +24,10 @@ private fun readNetworkFromCsv(fileName: String): Network {
         val carID = car.values.get(0)
         val id = carID.toInt()
 
-        val carStatus = car.values.get(1)
-        val status = carStatus.toBoolean()
-
-        val newCar = Car(id, status)
+        val carWantsToDriveAtHour = car.values.get(1)
+        val wannaDriveAt = carWantsToDriveAtHour.toInt()
+        // "wannaDriveAt" is Type Int...probably we have to change in class "Car" Type Boolean to Int ???
+        val newCar = Car(id, wannaDriveAt)
```

Comment:
This way you add a new car _per line_ in the CSV. Looking at `cars.csv` you would get 9 cars with ID `1`.